src/main/java/formflow/library/FormFlowController.java

```diff
@@ -78,11 +117,12 @@ protected static void throwNotFoundError(String flow, String screen, String mess
   }
 
   /**
-   * If the submission information exists in the HttpSession, find it in the db. If a submission is not found, create a new one.
+   * If the submission information exists in the {@link HttpSession}, find it in the db. If a submission is not found, create a new one.
    *
-   * @param httpSession The HttpSession to look in for information
+   * @param httpSession The {@link HttpSession}to look in for information
    * @param flow        The name of the flow to retrieve data about
-   * @return Submission A Submission object from the database or a new one if one was not found
+   * @exception ResponseStatusException Thrown when a {@code submission} is not found when {@code getSubmissionFromSession()} is called.
```

Comment:
This technically doesn't throw this exception. It catches it and promptly ignores it, but it doesn't throw it.   Folks reading java docs only care about things that ripple to outside of the method.